src/main/java/org/datavyu/plugins/ffmpeg/examples/SimpleMediaPlayerExample.java

```diff
@@ -56,6 +56,10 @@ public void keyPressed(KeyEvent e) {
                         System.out.println("Play");
                         mediaPlayer.play();
                         break;
+                    case KeyEvent.VK_S:
+                        System.out.println("Step Forward, with FPS : "  + mediaPlayer.getFps());
```

Comment:
Why do print fps when we step?  It's counter intuitive.

If you want to get the info about fps then log it at startup.